app/v1/models/response/retrieveTransactionDetails/TransactionItem.scala

```diff
@@ -42,14 +40,10 @@ object TransactionItem {
     taxPeriodTo <- (JsPath \ "taxPeriodTo").readNullable[String]
     originalAmount <- (JsPath \ "originalAmount").readNullable[BigDecimal]
     outstandingAmount <- (JsPath \ "outstandingAmount").readNullable[BigDecimal]
-    subItems <- (JsPath \ "items").readNullable[Seq[SubItem]].map(_.map(
-      _.filterNot(item => item == SubItem.empty || item.subItemId.isEmpty)
```

Comment:
I'm not sure why this filter has been removed, since the items can still be empty as per the newest DES spec